File: LaboratoryWork/Semester_3/Laboratory_2/GoLang/utils/menu/menu.go
package menu

import "fmt"

func StartMenu() {
	fmt.Println("Список программ:")
	fmt.Println("0. Exit")
	fmt.Println("1. Arithmetic expression")
	fmt.Println(`2. Interactions with set (для использования перезапустите программу и используйте следующий шаблон: go run ./cmd/main --file somethingFile.txt --query "COMMAND StructName Data")`)
	fmt.Println("3. Set partition")
	fmt.Println("4. Subarrays")
	fmt.Println("5. Check AVL-tree balance")
	fmt.Println("6. Convolution method")
	fmt.Println("7. LRU cache")
	fmt.Print("Выберите программу: ")
}

File: LaboratoryWork/Semester_3/Laboratory_2/C++/src/utils/menu.cpp
#include "../../include/menu.hpp"
#include <iostream>
#include <stdexcept>

using namespace std;

void startMenu() {
    cout << "Список программ:" << endl;
    cout << "0. Exit" << endl;
    cout << "1. Arighmteic expression" << endl;
    cout << "2. Interactions with set (для использования перезапустите программу и используйте следующий шаблон: ./main --file somethingFile.txt --query \"COMMAND StructName Data\")" << endl;
    cout << "3. Set partition" << endl;
    cout << "4. Subarrays" << endl;
    cout << "5. Check AVL-tree balance" << endl;
    cout << "6. Convolution method" << endl;
    cout << "7. LRU cach" << endl;
    cout << "Выберите программу: ";
}